dashboard/state_aggregator.py

```diff
@@ -595,6 +597,28 @@ def sort_func(entry):
             num_filtered=num_filtered,
         )
 
+    async def list_cluster_events(self, *, option: ListApiOptions) -> ListApiResponse:
+        result = []
+        all_events = await self._client.get_all_cluster_events()
+        for _, events in all_events.items():
+            for _, event in events.items():
+                event["time"] = str(datetime.utcfromtimestamp(int(event["timestamp"])))
```

Comment:
Is it necessary to append the formatted "time" as well? It seems redundant, but fine to keep if we do this elsewhere.

Reply:
I thought it could be done from the consumer side (as state API is the consumer of the event and format the timestamp). I can add it in the future if it is necessary!